schema/config-vm.json

```diff
@@ -0,0 +1,66 @@
+{
+    "vm": {
+        "description": "configuration for virtual-machine-based containers",
+        "id": "https://opencontainers.org/schema/bundle/vm",
+        "type": "object",
+        "required" : [
+            "kernel",
```

Comment:
This trailing comma [is invalid JSON][1].

[1]: https://travis-ci.org/opencontainers/runtime-spec/jobs/342293097#L584